### ISSUE TITLE
UI-router resolve in version 0.2.0

### ISSUE THREAD

**Issue by canac** [Urigo/angular-meteor-auth#3]
Is it possible to continue using angular-meteor-auth with ui-router's `resolve`? In angular-meteor-auth version 0.1.0, I could do this:

```js
resolve: {
  currentUser($auth) {
    return $auth.requireUser();
  },
},
```

Is there an equivalent in version 0.2.0? If not, would you consider adding that highly useful functionality back?

**Comment by Urigo** [comment]
@canac please try version `0.2.1` , I've fixed a few issues there and everything should work as expected now

**Comment by canac** [comment]
I am sure that you are aware of this because it is fixed in master, but v0.2.1 is throwing "$rootScope._throttledDigest is not a function" TypeError exceptions.

**Comment by Urigo** [comment]
@canac are you sure you are on `0.2.1` and not `0.2.0`?  it's the only bug I've fixed in the `0.2.1` version...

**Comment by canac** [comment]
I'm sorry, you're right. I must have either screwed something up in my environment or not carefully checked my `.meteor/versions` file. I'm not getting that exception anymore, so you can consider this issue resolved.
